airmozilla/main/templates/main/event.html

```diff
@@ -121,7 +121,13 @@ <h1 class="entry-title">
           <label for="tab-details">Details</label>
           <div class="content">
             <p>
-              Start time: {{ event.start_time | js_date }}
+              {% if event.location %}
+                  {% if event.location.name.startswith(settings.DEFAULT_PRERECORDED_LOCATION[0]) %}
+                      Uploaded: {{ event.start_time | js_date }}
+                  {% else %}
+                      Start time: {{ event.start_time | js_date }}
+                  {% endif %}
+              {% endif %}
```

Comment:
There is another situation we need to think about. When you upload a new video, using the new uploader it'll exist in the database **without** a location. E.g. `event.location==None`. But it might still have a `start_time`.

Also, this template code is starting to get complicated. I think you should add a new method to the `Event` model class. Call it "is_prerecorded" and it should look something like this:
```python
def is_prerecorded(self):
    if self.location:
        if self.location.name.startswith(settings.DEFAULT_PRE....:
            return True
        return False
    return True
```
If you could do that please, can you also review other places where the `settings.DEFAULT_PRERECORDED_LOCATION` is being used and if we can do some refactoring there.